__tests__/actionCreators/search.test.js

```diff
@@ -19,7 +19,15 @@ describe('fetchSearchResults', () => {
     const dispatch = jest.fn()
     await fetchSearchResults(query)(dispatch)
     expect(dispatch).toHaveBeenCalledTimes(2)
-    expect(dispatch).toBeCalledWith({ type: 'GET_SEARCH_RESULTS_STARTED', payload: { query: '*', queryFrom: undefined } })
-    expect(dispatch).toBeCalledWith({ type: 'SET_SEARCH_RESULTS', payload: { query: '*', searchResults: mockSearchResults.results, totalResults: mockSearchResults.totalHits } })
+    expect(dispatch).toBeCalledWith({ type: 'GET_SEARCH_RESULTS_STARTED', payload: { query: '*', queryFrom: 0 } })
+    expect(dispatch).toBeCalledWith({
+      type: 'SET_SEARCH_RESULTS',
+      payload: {
+        query: '*',
+        searchResults: mockSearchResults.results,
+        totalResults: mockSearchResults.totalHits,
+        startOfRange: 0,
```

Comment:
in a perfect world, we might want a test for a diff search results page than the first/default.  Maybe make a ticket for an integration test (ok if stuff is mocked ... just want to ensure we have hookups correct)

Reply:
There is a separate ticket for the integration test :-)